### PR TITLE
[test](p2) print error message when assert fails

### PR DESCRIPTION
**Pull request by kaijchen**
## Proposed changes

print error message when assert fails

regression-test/suites/load_p2/broker_load/test_broker_load.groovy

### CODE REVIEW

**Comment by kaijchen** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: regression-test/suites/load_p2/broker_load/test_broker_load.groovy [hunk 1]
```diff
@@ -343,7 +343,7 @@ suite("test_broker_load_p2", "p2") {
                     }
                     if (result[0][2].equals("CANCELLED")) {
```
Comment (liaoxin01) [inline regression-test/suites/load_p2/broker_load/test_broker_load.groovy:344]: logger.info("Load result: " + result[0])

Comment (kaijchen) [inline regression-test/suites/load_p2/broker_load/test_broker_load.groovy:344]:
```suggestion
                    if (result[0][2].equals("CANCELLED")) {
                        logger.info("Load result: " + result[0])
```

**Comment by kaijchen** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by Yukang-Lian — APPROVED** [review]

**Pull request merged**